test/unit/spec/common/jobManager-Spec.js

```diff
@@ -93,12 +130,9 @@ define([
     describe('the JobManager instance', () => {
         beforeEach(function () {
             this.model = Props.make({
-                data: {
-                    exec: {
-                        jobs: Jobs.jobArrayToIndexedObject(JobsData.allJobs),
-                    },
-                },
+                data: {},
             });
+            Jobs.populateModelFromJobArray(JobsData.allJobsWithBatchParent, this.model);
```

Comment:
The original set up didn't have the batch parent job (doh!) so had to be altered.